blockbase/theme.json

```diff
@@ -294,6 +294,11 @@
 				"opacity": 1,
 				"width": "150px"
 			},
+			"social-links": {
```

Comment:
This is just for testing

Reply:
this is only being applied to social links inside navigation blocks so maybe we'd better change the custom variable name to reflect that?

Reply:
good idea

Reply:
changed to navigaiton/social-links, what do you think?

Reply:
yeah that sounds about right!